src/wallet/crypter.h

```diff
@@ -250,7 +250,7 @@ class CCryptoKeyStore : public CBasicKeyStore
     bool HaveSaplingSpendingKey(const libzcash::SaplingFullViewingKey &fvk) const
```

Comment:
In `GetKeys` and `GetSproutPaymentAddresses`, it's necessary to lock before calling `IsCrypted()`.